conf/generator.yaml

```diff
@@ -145,3 +160,102 @@ modules:
       - artifactid: spring-boot-starter-test
         groupid: org.springframework.boot
         scope: test
+  - name: tracing
+    description: "Distributed tracing with OpenTracing"
+    guide_ref: ""
+    tags:
+      - tracing
+      - monitoring
+    dependencies:
+      - artifactid: opentracing-spring-jaeger-web-starter
+        groupid: io.opentracing.contrib
+      - artifactid: spring-boot-starter
+        groupid: org.springframework.boot
+      - artifactid: spring-boot-starter-test
+        groupid: org.springframework.boot
+        scope: test
+  - name: infinispan
+    description: "Remote Infinispan cache and datagrid"
+    guide_ref: ""
+    tags:
+      - persistence
+      - cache
```

Comment:
We need perhaps the info that it is a local or remote cache ?

Reply:
It is in the description already. What additional tag would you suggest?

Reply:
As the info is included within the title, then it will be displayed even if you select cache as topic and then the user could select the appropriate starter. No further modification is then needed